shared_model/backend/protobuf/query_responses/proto_account_asset_response.hpp

```diff
@@ -33,12 +30,9 @@ namespace shared_model {
           const override;
 
      private:
-      template <typename T>
-      using Lazy = detail::LazyInitializer<T>;
-
       const iroha::protocol::AccountAssetResponse &accountAssetResponse_;
 
-      const Lazy<std::vector<AccountAsset>> accountAssets_;
+      const std::vector<AccountAsset> accountAssets_;
```

Comment:
Please fix naming style. Snake case should be used instead. Fix `accountAssetResponse_` name too.